### PR TITLE
Add unique topics handling and fix rate calc

### PR DESCRIPTION
**Pull request by savex**
Bool flag unique-topics makes producers and
    consumers generate topics using counters in
    "topic-<num>" format.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -435,17 +447,19 @@ enum Commands {
     /// Creates a producer swarm
     Producers {
         #[clap(short, long)]
-        topic: String,
+        topic_prefix: String,
```
Comment (StephanDollberg) [inline src/main.rs]: Changing `--topic` to `--topic-prefix` arg probalby breaks a bunch of invocations in various places so it's not worth changing. 

Instead I'd either just keep `--topic` and only use `--unique_topics` or just add `--topic-prefix` and then require either `--topic` or `--topic-prefix`.

**Review by StephanDollberg — COMMENTED** [review]
It would be great to have two separate commits for the new arg and the rate fix. Not immediately clear what is wrong with the later.

File: src/main.rs (hunk 1 above)
Comment (savex) [reply src/main.rs]: Yep, I thought to update it to better reflect the content. But fixing all those in main repo is a pain. Let me update this.

**Comment by savex** [comment]
> It would be great to have two separate commits for the new arg and the rate fix. Not immediately clear what is wrong with the later.

When the `total_time` has very small value `as_secs()` will produce `0` seconds. And after that, comes division by zero in the next formula.

**Review by StephanDollberg — APPROVED** [review]

**Pull request merged**